scripts/classes.py

```diff
@@ -0,0 +1,96 @@
+#Scopes and Namespaces Example
+
+def scope_test():
```

Comment:
I see the exact code in [Python doc](https://docs.python.org/3/tutorial/classes.html). 
Please avoid copy-paste.

Reply:
okk